modified links in technical concepts (#1319)
SoftwareAG/c8y-docs: content/concepts/applications-bundle/overview.md

layout: bundle
---

>  {{< product-c8y-iot >}} applications can be of two types:

Reviewer: nitpicky, but sounds a bit like there could also be the reading "can be both".
maybe something like "... applications have one of the following types:"

>  {{< product-c8y-iot >}} applications can be of two types:

Author: ```suggestion
 There are two types of {{< product-c8y-iot >}} applications:
```
